.travis.yml

```diff
@@ -14,4 +14,7 @@ script:
 after_success:
   - yarn run test-integration
   - docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
-  - ./scripts/ci-deployment/docker_ci.sh
+  - bash ./scripts/ci-deployment/github/build-docker-images.sh
```

Comment:
Can these scripts be combined into a single one that calls them all (for terseness).